packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/launch-modal.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, DesktopWrapper, MobileWrapper, Modal, Text, UILoader } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import React from 'react';
+import './launch-modal.scss';
+import { observer, useStore } from '@deriv/stores';
+import LaunchModalChartImage from 'Assets/SvgComponents/launch/ic-chart-launch.svg';
+import LaunchModalChartImageDark from 'Assets/SvgComponents/launch/ic-chart-launch-dark.svg';
+
+type LaunchModalProps = {
+    handleChange: () => void;
+    open: boolean;
+};
+
+const ContinueButton = ({ handleOpen }: { handleOpen: () => void }) => (
+    <Modal.Footer>
+        <Button className='continue-button' has_effect onClick={handleOpen} primary large>
+            <Localize i18n_default_text='Continue' />
+        </Button>
+    </Modal.Footer>
+);
+
+const InfoDisplay = () => {
```

Comment:
@ahmadtaimoor-deriv Let's follow the best practice and place each child component into a separate file, and pass what they need via props. Please ensure that all of them are provided with test coverage by running npm run test:jest -- --collectCoverage  🙏🏻

Reply:
The components are broken down in smaller ones now . Regarding the test coverage it is also broken into file. Plus 
we will anyways be removing the modal in two weeks.